PostInstallCob4.sh

```diff
@@ -1,14 +1,16 @@
 #!/bin/bash
+set -e # force the script to exit if any error occurs
+
 #### COMMON PARAMETERS
 usage=$(cat <<"EOF"
 INFO: This script is a helper tool for the setup and installation of Care-O-bot: \n
-  1.  Create root ssh keys \n
-  2.  Synchronizes the passwords and robot user \n
-  3.  Setup the robot bashrc and workspace \n
-  4.  Add and setup the mimic user configuration \n
-  5.  Install Upstart software \n
-  6.  Update Upstart software \n
-  7.  Setup the udev rules for the scanners 
+1. Update root ssh keys\n
+2. Synchronize robot user\n
+3. Setup robot bashrc and Workspace\n
+4. Setup mimic user\n
+5. Install upstart\n
```

Comment:
for me make more sense separate both functions Install and Update, but should be also ok

Reply:
calling Install multiple times is possible yo you can simply update with calling Insall upstart again. No need to maintain two functions